docs/modules/ROOT/pages/getting-started/connecting-to-neo4j.adoc

```diff
@@ -0,0 +1,119 @@
+[[connecting-to-neo4j]]
+:description: This tutorial demonstrates how to connect to Neo4j and utilize Cypher Builder.
+= Connecting to Neo4j
+
+The Cypher Builder library helps you compose Cypher Queries, but sending these to Neo4j must be done separately through the link:https://www.npmjs.com/package/neo4j-driver[Neo4j Driver].
+
+
+== Setup the project
+
+Initialize your project by following the instructions in xref:/getting-started/installation.adoc[Installation].
+
+To connect to Neo4j, make sure to have both `@neo4j/cypher-builder` and `neo4j-driver` installed in your NodeJS project:
+
+[source, cmd]
+----
+npm install @neo4j/cypher-builder neo4j-driver
+----
+
+For this example, the Movies dataset is used. Alternatively, you can initialize the dataset with the following Cypher examples:
+
+[source, cypher]
+----
+CREATE(:Movie {title: "The Matrix"})
+CREATE(:Movie {title: "The Terminal"})
+----
+
+== Initialize neo4j-driver
+In your JavaScript file, initialize the driver following the instructions in the link:https://neo4j.com/docs/javascript-manual/current/[JavaScript Driver Manual]:
+
+
+[source, javascript]
+----
+import Cypher from "@neo4j/cypher-builder";
+import neo4j from "neo4j-driver";
+
+const driver = neo4j.driver("neo4j://localhost", neo4j.auth.basic("neo4j", "password"));
+----
+
+== Construct your Cypher query
+
+Compose your desired query using the Cypher object. For instance:
+
+[source, javascript]
+----
+const movie = new Cypher.Node({ labels: ["Movie"] });
+const query = new Cypher.Match(movie).return([movie.property("title"), "title"]);
+----
+
+This query will translate to:
+
+[source, cypher]
+----
+MATCH (this0:Movie)
+RETURN this0.title AS title
+----
+
+== Execute the query
+
+Use `.build` on the query object to generate the Cypher query string and the necessary parameters to pass to `neo4j-driver`
+
+
+[source, javascript]
+----
+const { cypher, params } = query.build();
+const { records } = await driver.executeQuery(cypher, params);
+----
+
+Close the driver connection:
+
+[source, javascript]
+----
+await driver.close();
+----
+
+
+== Consume the results
+
+The `records` object can be consumed as explained in the link:https://neo4j.com/docs/javascript-manual/current/[JavaScript Driver Manual]:
+
+[source, javascript]
+----
+for (const record of records) {
+    console.log(record.get("title"));
+}
+----
+
+With the example data above, this should output:
```

Comment:
If we change the previous title to "Initialize the driver", then the link should be like that:

```suggestion
With the xref:getting-started/connecting-to-neo4j.adoc#_initialize_the_driver[previous example data], this should output:
```